constants.js

```diff
@@ -0,0 +1 @@
+export const MERMAID_BLOCK_LANG = 'mermaid-render-svg';
```

Comment:
Can we just call it `mermaid` like github has? Then stuff that works on github will just work on us.

https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

Reply:
I'm reluctant, since it would conflict with using `mermaid` syntax highlighting? But now that I see that's not part of our set of langs, sure!